index.js

```diff
@@ -69,20 +72,6 @@ S3Adapter.SCHEMA_FIELD_DEFAULTS = {
 	etag: false,
 };
 
-// Return a knox client configured to interact with the specified file.
-S3Adapter.prototype._knoxForFile = function (file) {
-	// Clients are allowed to store the bucket name in the file structure. If they
-	// do it'll make it possible to have some files in one bucket and some files
-	// in another bucket. The knox client is configured per-bucket, so if you're
-	// using multiple buckets we'll need a different knox client for each file.
-	if (file.bucket && file.bucket !== this.options.bucket) {
-		var s3options = assign({}, this.options, { bucket: file.bucket });
-		return knox.createClient(s3options);
-	} else {
-		return this.client;
-	}
-};
```

Comment:
the S3 client can handle multiple bucket names. but anyway for this instance, we should be using only `this.options.bucket`. Should it ever happen that the `file.bucket` and `options.bucket` differ? wheN?